src/applications/appeals/995/sass/995-supplemental-claim.scss

```diff
@@ -5,3 +5,7 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../../platform/forms/sass/m-form-confirmation";
+
+#root_form5103Acknowledged-label {
```

Comment:
It's unfortunate that this isn't taken care of already for you by the form system